### ISSUE TITLE
Assessment of metric prediction issues

### ISSUE THREAD

**Issue by liudong315** [likyoo/Siam-NestedUNet#8]
In the eval.py file, an error is reported, how should I fix it, thanks!
![image](https://user-images.githubusercontent.com/33924595/119282164-2b5d7600-bc6b-11eb-9211-61f370963d79.png)

**Comment by likyoo** [comment]
I think you can debug it and check the contents of the labels and cd_preds.
Here is an official sample from [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html):
```
>>> tn, fp, fn, tp = confusion_matrix([0, 1, 0, 1], [1, 1, 1, 0]).ravel()
>>> (tn, fp, fn, tp)
(0, 2, 1, 1)
```
